dao/src/main/java/greencity/entity/RatingPoints.java

```diff
@@ -0,0 +1,30 @@
+package greencity.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import jakarta.persistence.Id;
+import jakarta.persistence.Column;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "rating_points")
+public class RatingPoints {
```

Comment:
please use `record`

Reply:
But it is an `Entity`. `Entity` can't be a `record`, can it?